uint32 seq
time stamp
string frame_id
float64 ult_right
float64 ult_left
float64 ir_right
float64 ir_left
float64 compass_x
float64 compass_y
float64 compass_z
uint64 servo_angle
